cumulus/tasks/discover-pdrs/package.json

```diff
@@ -37,6 +37,10 @@
   "author": "Cumulus Authors",
   "license": "Apache-2.0",
   "dependencies": {
+    "@cumulus/common": "^1.0.0-1.0.0-beta.9.1",
+    "@cumulus/cumulus-message-adapter-js": "~0.0.1-beta.2",
```

Comment:
The disadvantage of not having `@cumulus/cumulus-message-adapter-js` in the cumulus repo is that everytime, we make a change there, we have to go manually update the version number in 10s of tasks.

Reply:
Shouldn't having the ~ in front auto update for minor changes?

Reply:
yeah, that's definitely a good strategy. Although, for someone reason it didn't work us yesterday.

Reply:
The reason that we created a wrapper for the executable message adapter is so that we would be able to update the message protocol _without_ having to update the version of `cumulus-message-adapter-js`.  That's the sole purpose of the sled/cumulus-message-parser.

Reply:
Agreed! I think Lauren's suggestion is the best course (we should use `~` instead of `^`)

Reply:
Actually, it seems that it's the other way 😕 

> In the simplest terms, the tilde matches the most recent minor version (the middle number). ~1.2.3 will match all 1.2.x versions but will miss 1.3.0.

> The caret, on the other hand, is more relaxed. It will update you to the most recent major version (the first number). ^1.2.3 will match any 1.x.x release including 1.3.0, but will hold off on 2.0.0.